db/migrate/20160330000130_add_foreign_key_constraints.rb

```diff
@@ -0,0 +1,33 @@
+class AddForeignKeyConstraints < ActiveRecord::Migration
+  def change
+    add_foreign_key :absences, :student_school_years
```

Comment:
Agreed. This wouldn't be an across the board change you'd want to make for all instances of a foreign key so as someone who doesn't know all of the business logic yet I guess I'd just base the addition on whether I see a belongs_to relationship or a presence validation.....